### PR TITLE
manifest: openthread: regular openthread upmerge `37fb770`

### PR DESCRIPTION
**Pull request by maciejbaczmanski**
Regular OpenThread update to `37fb770`

Signed-off-by: Maciej Baczmanski <maciej.baczmanski@nordicsemi.no>

### CODE REVIEW

**Review by mariuszpos — APPROVED** [review]

**Comment by rlubos** [comment]
@maciejbaczmanski Can you rebase? Downstream jobs will fail due to conflict.

**Review by edmont — APPROVED** [review]

**Comment by rlubos** [comment]
@maciejbaczmanski The CI failures are due to yet another PR being merged in the morning (https://github.com/nrfconnect/sdk-zephyr/pull/1218), I wasn't aware it was merged. So in order to sort this out, we need yet another rebase (I've updated the manifest just now).

**Comment by maciejbaczmanski** [comment]
> @maciejbaczmanski The CI failures are due to yet another PR being merged in the morning ([nrfconnect/sdk-zephyr#1218](https://github.com/nrfconnect/sdk-zephyr/pull/1218)), I wasn't aware it was merged. So in order to sort this out, we need yet another rebase (I've updated the manifest just now).

Rebased. Thanks for keeping an eye on that

**Review by pjasinski990 — APPROVED** [review]

**Comment by rlubos** [comment]
@maciejbaczmanski It seems that something might be wrong with this Thread update? Thread tests failed three times in a row in the same way, so I start to suspect it's not just test instability.

**Comment by maciejbaczmanski** [comment]
> @maciejbaczmanski It seems that something might be wrong with this Thread update? Thread tests failed three times in a row in the same way, so I start to suspect it's not just test instability.

Yes, I've just found the problem.  -DOT_OPERATIONAL_DATASET_AUTO_INIT=ON while building posix application is missing. I'm working on a fix for that

**Review by canisLupus1313 — APPROVED** [review]

**Comment by maciejbaczmanski** [comment]
Rebased. Issue has been fixed in tests implementation

File: west.yml [hunk 1]
```diff
@@ -59,7 +59,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 35510c3d8e95826803ffaee1bb678163e1846320
+      revision: pull/1233/head
```
Comment (rlubos) [inline west.yml:62]: `eb28632680089351c7bd4365cf3742bf7eff70d3`

**Review by rlubos — APPROVED** [review]

**Pull request merged**